x/liquiditypool/keeper/msg_server_create_pool.go

```diff
@@ -38,6 +59,37 @@ func (k msgServer) CreatePool(ctx context.Context, msg *types.MsgCreatePool) (*t
 		return nil, errorsmod.Wrap(err, "invalid base offset")
 	}
 
+	if baseOffset.IsPositive() {
+		return nil, errorsmod.Wrap(err, "base offset must be in range (-1, 0]")
```

Comment:
@kimurayu45z Should the baseOffset be between [-1, 1]?

Reply:
if `base_offset = 0.5`

$$\text{price}(\text{tick}) = 1.0001 ^ {\text{tick} - 0.5}$$

so, 

$$\text{price}(\text{tick}) = 1.0001 ^ {\text{tick}} \times 1.0001 ^ {-0.5} \approx 1.0001 ^ {\text{tick}} \times 0.99995$$

On th other hand, if `base_offset = -0.5`

$$\text{price}(\text{tick}) = 1.0001 ^ {\text{tick} - (-0.5)} = 1.0001 ^ {\text{tick} + 0.5}$$

so,

$$\text{price}(\text{tick}) = 1.0001 ^ {\text{tick}} \times 1.0001 ^ {0.5} \approx 1.0001 ^ {\text{tick}} \times 1.00005$$

For stabled coins, base_offset should be 0.5 if it is desirable not to exceed 1.

Reply:
I would suggest `-1 < base_offset < 1` for handling in msgServer.